taipy/gui/builder/_element.py

```diff
@@ -37,30 +40,63 @@ def __new__(cls, *args, **kwargs):
         return obj
 
     def __init__(self, *args, **kwargs):
+        source_frame = inspect.stack()[3]
+        self.__locals = {
+            key: value
+            for key, value in t.cast(FrameType, source_frame.frame).f_locals.items()
+            if not key.startswith("__")
```

Comment:
Do we need to care about variables starting with a single '_'?

Reply:
Just want to filter out the double underscore / private variable so we can go through less variables when processing

Reply:
Yes I understand that.
My question can be shorten to: why not '`if not key.startswith("_")`'